controller/cmd/proxy-injector/main.go

```diff
@@ -40,7 +41,12 @@ func main() {
 		log.Fatalf("failed to mount the ca bundle: %s", err)
 	}
 
-	webhookConfig, err := injector.NewWebhookConfig(k8sClient, *controllerNamespace, *webhookServiceName, k8sPkg.MountPathTLSTrustAnchor)
+	rootCA, err := ca.NewCA()
```

Comment:
👍 to @olix0r's suggesting of moving the `CA` struct and this func to `pkg/tls` or somewhere similar.

Note that there's already a very small `pkg/tls` package that provides the `PEMEncodeCert` and `PEMEncodeKey` funcs, and incidentally those functions are called from the `tlsConfig` function in controller/proxy-injector/server.go, which you're modifying as part of this branch. I think it would make sense to consolidate `controller/ca/ca.go` and `pkg/tls` into a single package as part of this change.